.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I would prefer not to include this file in the repository, as it may conflict with the contributor's local settings.

Instead is it possible to [use a marker to omit from the toc](https://github.com/taiki-e/cargo-llvm-cov/blob/8a3553b86551eabf9c30c060b1f72a5bbccb98c6/README.md?plain=1#L601)?